lms/envs/common.py

```diff
@@ -3186,13 +3168,15 @@ def _make_locale_paths(settings):
     'JWT_EXPIRATION': 30,
     'JWT_SUPPORTED_VERSION': '1.1.0',
 
-    'JWT_SECRET_KEY': DEFAULT_JWT_ISSUER['SECRET_KEY'],
-    'JWT_ISSUER': DEFAULT_JWT_ISSUER['ISSUER'],
-    'JWT_AUDIENCE': DEFAULT_JWT_ISSUER['AUDIENCE'],
-    'JWT_ISSUERS': [
-        DEFAULT_JWT_ISSUER,
-        RESTRICTED_APPLICATION_JWT_ISSUER,
-    ],
+    'JWT_ALGORITHM': 'HS256',
+    'JWT_SECRET_KEY': SECRET_KEY,
+
+    'JWT_SIGNING_ALGORITHM': 'RS512',
+    'JWT_PRIVATE_SIGNING_JWK': 'change-me',
+    'JWT_PUBLIC_SIGNING_JWK_SET': 'change-me',
```

Comment:
should these json strings default to normal text here (and over in configuration) or is that more/less likely to fail weirdly.

Reply:
Good point.  I changed it to default to `None`.